tutorials/image/cifar10_estimator/README.md

```diff
@@ -33,10 +33,10 @@ $ python cifar10_main.py --data_dir=/prefix/to/downloaded/data/cifar-10-batches-
                          --num_gpus=0 \
                          --train_steps=1000
 
-# Run the model on CPU and 2 CPUs. After training, it runs the evaluation.
+# Run the model on CPU and 2 GPUs. After training, it runs the evaluation.
 $ python cifar10_main.py --data_dir=/prefix/to/downloaded/data/cifar-10-batches-bin \
                          --model_dir=/tmp/cifar10 \
-                         --is_cpu_ps=False \
+                         --is_cpu_ps=True \
```

Comment:
If I get it right in the comment it says that is using is_cpu_ps=True: "# Run the model on CPU and 2 GPUs. After training, it runs the evaluation." Also, in my experiments is_cpu_ps=True was faster ( but running with experiments and an older version of the code)